src/edu/csus/ecs/pc2/exports/ccs/ResultRow.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 1989-2023 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.exports.ccs;
+
+import edu.csus.ecs.pc2.core.Constants;
+import edu.csus.ecs.pc2.core.Utilities;
+
+public class ResultRow {
```

Comment:
This class should contain a header description of what it is for.  Otherwise, the rest of us are forced to slug through the code to try to figure out what the purpose of the class is.

Reply:
comment removed